test/studies/comd/llnl/CoMD.prediff

```diff
@@ -6,17 +6,36 @@ test_name = sys.argv[1]
 out_file  = sys.argv[2]
 tmp_file  = out_file + ".prediff.tmp"
 
+EPSILON = 1e-6
+vals = {}
+vals['   Initial energy  :'] = -1.166063303477
+vals['   Final energy    :'] = -1.166049767266
+vals['   eFinal/eInitial :'] = 0.999988
+
+def verify(line):
+    info = line.split(' ')
+    pre = ' '.join(info[:-1])
+    if pre in vals:
+        last = float(info[-1])
+        if abs(last - vals[pre]) < EPSILON:
```

Comment:
Ah, interesting. I don't think we can rely on python 3 for start_test yet though 😢